lib/apparmortest.pm

```diff
@@ -802,6 +802,8 @@ Run post_fail_hook and upload audit logs
 sub post_fail_hook {
     my ($self) = shift;
 
+    return if (get_var('NOLOGS'));
+
```

Comment:
Please use just `return if get_var('NOLOGS')` without the additional pair of parentheses as there is only a single argument.

Reply:
Thanks @okurz 
fixed as your kindly suggestion.